x-pack/auditbeat/module/system/package/rpm_linux.go

```diff
@@ -0,0 +1,302 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build linux
```

Comment:
`linux,cgo` better reflects the requirements of the file. Likewise anywhere that uses a tag that's meant to the opposite should updated (I think that would be `// +build !linux !cgo`).

You'll want to update anything that depends on this file directly to include the same label. This should allow for `CGO_ENABLED=0 go build` to still work on Linux.

Reply:
Thanks, should be done with the latest commit. I tested `CGO_ENABLED=0 go build` on linux.